investments/admin.py

```diff
@@ -0,0 +1,14 @@
+from django.contrib import admin
+from investments.models import Subscription
+
+
+@admin.register(Subscription)
+class SubscriptionAdmin(admin.ModelAdmin):
+    """
+    Admin configuration for the Subscription model.
+
+    Makes the 'investment_share' field read-only to prevent manual edits,
+    and displays key fields in the admin list view.
+    """
+    readonly_fields = ('investment_share',)
```

Comment:
👍  Marking `investment_share` as read-only in the admin is a good practice since it's calculated automatically.